tests/integration/REST/GetContentTreeChildrenTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace Ibexa\Tests\Integration\AdminUi\REST;
+
+use Ibexa\Contracts\Test\Rest\Request\Value\EndpointRequestDefinition;
+
+final class GetContentTreeChildrenTest extends BaseAdminUiRestWebTestCase
+{
+    private const ENDPOINT_CONTENT_TREE_CHILDREN_URL = 'api/ibexa/v2/location/tree/load-subitems/1/10/0';
+    private const RESOURCE_TYPE = 'ContentTreeNode';
+    private const SNAPSHOT_FILTER_PATH = self::RESOURCE_TYPE . '/filter/%s/%s';
+    private const SNAPSHOT_NO_FILTER = 'no-filter';
+    private const LOCATION_ID_MEDIA = 43;
+
+    protected static function getEndpointsToTest(): iterable
```

Comment:
Missing docblock for `iterable`.

Reply:
Docblock is added to parent class: https://github.com/ibexa/test-rest/blob/main/src/contracts/BaseRestWebTestCase.php#L28